concepts/src/controllers/healthcheck.ts

```diff
@@ -0,0 +1,16 @@
+import { RequestHandler } from "express";
+import asyncHandler from "express-async-handler";
+import { Config } from "../../config";
+
+type PathParams = { id: string };
+
+const healthcheckController = (config: Config): RequestHandler<PathParams> => {
+  return asyncHandler(async (req, res) => {
+    res.status(200).json({
+      status: "ok",
+      config,
```

Comment:
When I saw this, I was slightly concerned that it might be a vector for leaking secrets.

It isn't, but is there a risk that someone might unwittingly expose something in the future by adding it to the config?